bin/citrea/src/test_rpc.rs

```diff
@@ -153,6 +153,7 @@ fn regular_test_helper(payload: serde_json::Value, expected: &serde_json::Value)
             ],
             phantom_data: PhantomData,
             pub_key: vec![],
+            deposit_data: vec![],
```

Comment:
put some data here to test the serialization

the serializaion should be as follows


[
  "0x123123",
  "0x123123",
]